tests/sawtooth-pbft-test.dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Intel Corporation
+# Copyright 2018 Bitwise IO, Inc.
```

Comment:
If this file started off as a copy of an intel file it should include both copyrights so it looks like:

Copyright 2018 Intel Corporation
Copyright 2018 Bitwise IO, Inc.